examples/go/go.mod

```diff
@@ -1,11 +1,9 @@
-module github.com/twilio/twilio-go
+module github.com/twilio/twilio-oai-generator
 
 go 1.16
 
 require (
-	github.com/golangci/golangci-lint v1.37.0 // indirect
-	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/tools v0.1.0 // indirect
 	github.com/golang/mock v1.5.0
+	github.com/stretchr/testify v1.7.0
+	github.com/twilio/twilio-go v0.6.0
```

Comment:
After the next version of twilio-go is released you can update the version by running `go get github.com/twilio/twilio-go@latest`

Reply:
Can we either automate that or have this point to the latest? My concern is if we have, for example, 2 releases since this file was updated, it will not be obvious we need to run that command to test against the current version.

Reply:
I've changed the dependency to point to the last commit on the main branch of twilio-go.

Reply:
In that case, the travis build shouldn't be failing.

Reply:
Looks like the failure had something todo with the way we were copying the generated code. I updated the generate command to generate directly inside the examples/go folder